CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Improvements
+
+- Add permissions for `INTERNET` and `ACCESS_NETWORK_STATE` in the AndroidManifest.xml of the Flutter example app. ([#2255](https://github.com/getsentry/sentry-dart/pull/2255))
```

Comment:
for sample apps, or internal things that the user doesn't directly affect we don't have to add changelog.

you can add #skip-changelog to the issue description

Reply:
thanks, for the hint. I wasn't aware of this. should I remove the changelog entry?

Reply:
I've noticed this while making another fix myself and removed in #2260